src/applications/search/containers/SearchApp.jsx

```diff
@@ -82,7 +82,9 @@ class SearchApp extends React.Component {
     const hasNewResults =
       prevProps.search.loading && !this.props.search.loading;
 
-    if (hasNewResults) {
+    if (this.state.landedWithSearchInput && this.searchFieldRef?.current) {
```

Comment:
Latest changes: to temporarily work around an issue with the search typeahead web component ([see this Slack thread for details](https://dsva.slack.com/archives/C01DBGX4P45/p1712227557801769)), we needed to force focus to the search input when the page is loaded with a search term prefilled. This will help the typeahead component dismiss itself (it otherwise gets stuck open).